packages/firebase_auth/firebase_auth_platform_interface/lib/src/method_channel/utils/pigeon_helper.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:firebase_auth_platform_interface/firebase_auth_platform_interface.dart';
+import 'package:firebase_auth_platform_interface/src/pigeon/messages.pigeon.dart';
+
+List<MultiFactorInfo> multiFactorInfoPigeonToObject(
+  List<PigeonMultiFactorInfo?> pigeonMultiFactorInfo,
+) {
+  return pigeonMultiFactorInfo
+      .where((element) => element != null)
```

Comment:
use package:collection and its `whereNotNull`

Reply:
Nice